### DISCUSSION TITLE
Percy CLI no longer sends snapshots while it's taking them

### DISCUSSION THREAD

**Discussion by patricknelson** [percy/cli#906] [Support]
## The problem

Percy CLI v1.1.0 no longer sends snapshots while it's taking them. Why? I have roughly 72 snapshots I need to take and sending them while it's taking them (like the beta version[s] did, e.g. 1.0.0-beta.76) was super helpful and speed up testing and iteration.

## Environment

- Node version: 16.4.2
- `@percy/cli` version: 1.1.0
- Version of Percy SDK you’re using:
- If needed, a build or snapshot ID:
- OS version: Windows 10
- Type of shell command-line [interface]: bash (via Windows Terminal)

## Details

n/a

## Debug logs

n/a

## Code to reproduce issue

n/a

**Comment by Robdel12** [comment]
Hey @patricknelson! I'm not sure I understand what you're saying here. Can you share full logs that illustrates the problem you're seeing? We'll need more details to help

**Reply by patricknelson** [reply]
Apologies for the confusion here. This is a _fairly minor_ optimization but I did like it. Unfortunately I don't have logs but I can still try to retrieve those. I think maybe it just turns out there's an incorrect perception on my part. 

Essentially what I noticed was that:

1. `@percy/cli@1.0.0-beta.76` began uploading my snapshots _immediately as_ they were being taken locally. The front-end UI on percy.io therefore immediately resulted in "Receiving build" as soon as I started the build in the command line.
2. `@percy/cli@1.1.0` didn't appear to begin uploading any of my snapshots until all 72 of them were taken. It was only at that point (after a minute or two delay) at which it went into "Receiving build"
 
However, that said, I haven't been able to precisely time the start of each command and compare that to the actual end of the build (since my builds are small with just two screenshots per snapshot). I just had what was probably an incorrect perception that _maybe_ my builds were effectively finished earlier (even with the same total build time), since it _looked_ like maybe the "Processing snapshots" (or if you click, "Processing visual changes") appeared to have a head start in the beta version.

**Reply by patricknelson** [reply]
Here's a quick side-by-side demo of what I'm referring to:

**@percy/cli@1.0.0-beta.76**
![1 0 0-beta 76](https://user-images.githubusercontent.com/4269377/166079093-311871ab-2df4-4517-b511-00df7d9761c0.gif)

**@percy/cli@1.1.0**
![1 1 0](https://user-images.githubusercontent.com/4269377/166079094-5c3bf208-7aeb-4938-ac56-6db3e5ae39ad.gif)
